pkg/koordlet/runtimehooks/nri/server.go

```diff
@@ -218,7 +218,7 @@ func (p *NriServer) CreateContainer(pod *api.PodSandbox, container *api.Containe
 	return adjust, nil, nil
 }
 
-func (p *NriServer) UpdateContainer(pod *api.PodSandbox, container *api.Container) ([]*api.ContainerUpdate, error) {
+func (p *NriServer) UpdateContainer(_ context.Context, pod *api.PodSandbox, container *api.Container, r *api.LinuxResources) ([]*api.ContainerUpdate, error) {
```

Comment:
What if the new argument `r != nil`, does it override the `container`?

Reply:
No, it depends on what you will do with r. If you leave it alone, it will do nothing, but at the other side, you can update container resource based on the paratmeters.

Reply:
ok